soccer/gameplay/root_play.py

```diff
@@ -192,7 +195,7 @@ def goalie_id(self):
 
     @goalie_id.setter
     def goalie_id(self, value):
-        self._goalie_id = None if value == -1 else value
+        self._goalie_id = value
```

Comment:
@justbuchanan @jgkamat It may take me a little bit longer to remember how all this works, its been a month since I did this and the internet here is having some issues. I will do what I can but I might not be able to get back to you with 100% confidence until sometime on Monday.